updating/installing-update-service.adoc

```diff
@@ -51,6 +51,12 @@ include::modules/update-service-create-service-web-console.adoc[leveloffset=+2]
 
 include::modules/update-service-create-service-cli.adoc[leveloffset=+2]
 
+[NOTE]
+====
+For the policy engine route if you see `ReconcileCompleted` status as `false`  with reason `CreateRouteFailed` caused by `host must conform to DNS 1123 naming convention`
```

Comment:
s/route if/route, if

s/with reason/with the reason

Remove line break between `host must conform to DNS 1123 naming convention` and `must be no more than 63 characters`.

Reply:
Resolved in https://github.com/openshift/openshift-docs/commit/5fe3272940e4c3b8ee27d792eb79dd78cde30345